From: Andreas Rheinhardt <andreas.rheinhardt@outlook.com>
Date: Mon, 1 Aug 2022 14:41:38 +0200
Subject: fate/lavf-image: Disable file checksums for exr tests

The generated files are endian-dependent, so no checksums
may be part of the ref files.

Fixes ticket #9854.

Tested-by: Sebastian Ramacher <sramacher@debian.org>
Signed-off-by: Andreas Rheinhardt <andreas.rheinhardt@outlook.com>
(cherry picked from commit fe211aebbf2988f7a6588cf9f45277ca74b92763)
---
 tests/fate/lavf-image.mak           | 24 ++++++++++++------------
 tests/ref/lavf/none.gbrapf32le.exr  |  2 --
 tests/ref/lavf/none.gbrpf32le.exr   |  2 --
 tests/ref/lavf/none.grayf32le.exr   |  2 --
 tests/ref/lavf/rle.gbrapf32le.exr   |  2 --
 tests/ref/lavf/rle.gbrpf32le.exr    |  2 --
 tests/ref/lavf/rle.grayf32le.exr    |  2 --
 tests/ref/lavf/zip1.gbrapf32le.exr  |  2 --
 tests/ref/lavf/zip1.gbrpf32le.exr   |  2 --
 tests/ref/lavf/zip1.grayf32le.exr   |  2 --
 tests/ref/lavf/zip16.gbrapf32le.exr |  2 --
 tests/ref/lavf/zip16.gbrpf32le.exr  |  2 --
 tests/ref/lavf/zip16.grayf32le.exr  |  2 --
 13 files changed, 12 insertions(+), 36 deletions(-)

diff --git a/tests/fate/lavf-image.mak b/tests/fate/lavf-image.mak
index 38d3689..6d61233 100644
--- a/tests/fate/lavf-image.mak
+++ b/tests/fate/lavf-image.mak
@@ -59,18 +59,18 @@ $(FATE_LAVF_IMAGES): CMD = lavf_image
 $(FATE_LAVF_IMAGES): REF = $(SRC_PATH)/tests/ref/lavf/$(@:fate-lavf-%=%)
 $(FATE_LAVF_IMAGES): $(VREF)
 
-fate-lavf-none.grayf32le.exr:   CMD = lavf_image "-compression none  -pix_fmt grayf32le"
-fate-lavf-rle.grayf32le.exr:    CMD = lavf_image "-compression rle   -pix_fmt grayf32le"
-fate-lavf-zip1.grayf32le.exr:   CMD = lavf_image "-compression zip1  -pix_fmt grayf32le"
-fate-lavf-zip16.grayf32le.exr:  CMD = lavf_image "-compression zip16 -pix_fmt grayf32le"
-fate-lavf-none.gbrpf32le.exr:   CMD = lavf_image "-compression none  -pix_fmt gbrpf32le"
-fate-lavf-rle.gbrpf32le.exr:    CMD = lavf_image "-compression rle   -pix_fmt gbrpf32le"
-fate-lavf-zip1.gbrpf32le.exr:   CMD = lavf_image "-compression zip1  -pix_fmt gbrpf32le"
-fate-lavf-zip16.gbrpf32le.exr:  CMD = lavf_image "-compression zip16 -pix_fmt gbrpf32le"
-fate-lavf-none.gbrapf32le.exr:  CMD = lavf_image "-compression none  -pix_fmt gbrapf32le"
-fate-lavf-rle.gbrapf32le.exr:   CMD = lavf_image "-compression rle   -pix_fmt gbrapf32le"
-fate-lavf-zip1.gbrapf32le.exr:  CMD = lavf_image "-compression zip1  -pix_fmt gbrapf32le"
-fate-lavf-zip16.gbrapf32le.exr: CMD = lavf_image "-compression zip16 -pix_fmt gbrapf32le"
+fate-lavf-none.grayf32le.exr:   CMD = lavf_image "-compression none  -pix_fmt grayf32le" "" "no_file_checksums"
+fate-lavf-rle.grayf32le.exr:    CMD = lavf_image "-compression rle   -pix_fmt grayf32le" "" "no_file_checksums"
+fate-lavf-zip1.grayf32le.exr:   CMD = lavf_image "-compression zip1  -pix_fmt grayf32le" "" "no_file_checksums"
+fate-lavf-zip16.grayf32le.exr:  CMD = lavf_image "-compression zip16 -pix_fmt grayf32le" "" "no_file_checksums"
+fate-lavf-none.gbrpf32le.exr:   CMD = lavf_image "-compression none  -pix_fmt gbrpf32le" "" "no_file_checksums"
+fate-lavf-rle.gbrpf32le.exr:    CMD = lavf_image "-compression rle   -pix_fmt gbrpf32le" "" "no_file_checksums"
+fate-lavf-zip1.gbrpf32le.exr:   CMD = lavf_image "-compression zip1  -pix_fmt gbrpf32le" "" "no_file_checksums"
+fate-lavf-zip16.gbrpf32le.exr:  CMD = lavf_image "-compression zip16 -pix_fmt gbrpf32le" "" "no_file_checksums"
+fate-lavf-none.gbrapf32le.exr:  CMD = lavf_image "-compression none  -pix_fmt gbrapf32le" "" "no_file_checksums"
+fate-lavf-rle.gbrapf32le.exr:   CMD = lavf_image "-compression rle   -pix_fmt gbrapf32le" "" "no_file_checksums"
+fate-lavf-zip1.gbrapf32le.exr:  CMD = lavf_image "-compression zip1  -pix_fmt gbrapf32le" "" "no_file_checksums"
+fate-lavf-zip16.gbrapf32le.exr: CMD = lavf_image "-compression zip16 -pix_fmt gbrapf32le" "" "no_file_checksums"
 fate-lavf-jpg: CMD = lavf_image "-pix_fmt yuvj420p"
 fate-lavf-tiff: CMD = lavf_image "-pix_fmt rgb24"
 fate-lavf-gbrp10le.dpx: CMD = lavf_image "-pix_fmt gbrp10le" "-pix_fmt gbrp10le"
diff --git a/tests/ref/lavf/none.gbrapf32le.exr b/tests/ref/lavf/none.gbrapf32le.exr
index 099f36c..e365d8d 100644
--- a/tests/ref/lavf/none.gbrapf32le.exr
+++ b/tests/ref/lavf/none.gbrapf32le.exr
@@ -1,3 +1 @@
-c586035e67f9ba7f2a3777933b5b22a0 *tests/data/images/none.gbrapf32le.exr/02.none.gbrapf32le.exr
-1627003 tests/data/images/none.gbrapf32le.exr/02.none.gbrapf32le.exr
 tests/data/images/none.gbrapf32le.exr/%02d.none.gbrapf32le.exr CRC=0x068aca4e
diff --git a/tests/ref/lavf/none.gbrpf32le.exr b/tests/ref/lavf/none.gbrpf32le.exr
index 7e0b2bd..996cd26 100644
--- a/tests/ref/lavf/none.gbrpf32le.exr
+++ b/tests/ref/lavf/none.gbrpf32le.exr
@@ -1,3 +1 @@
-10f42423c6585fe5053c5457fba7b235 *tests/data/images/none.gbrpf32le.exr/02.none.gbrpf32le.exr
-1221481 tests/data/images/none.gbrpf32le.exr/02.none.gbrpf32le.exr
 tests/data/images/none.gbrpf32le.exr/%02d.none.gbrpf32le.exr CRC=0x95e1053f
diff --git a/tests/ref/lavf/none.grayf32le.exr b/tests/ref/lavf/none.grayf32le.exr
index 4d5fad8..9db5bb7 100644
--- a/tests/ref/lavf/none.grayf32le.exr
+++ b/tests/ref/lavf/none.grayf32le.exr
@@ -1,3 +1 @@
-8aa28b10bf2591b7030b78bc29907293 *tests/data/images/none.grayf32le.exr/02.none.grayf32le.exr
-410437 tests/data/images/none.grayf32le.exr/02.none.grayf32le.exr
 tests/data/images/none.grayf32le.exr/%02d.none.grayf32le.exr CRC=0x5443a443
diff --git a/tests/ref/lavf/rle.gbrapf32le.exr b/tests/ref/lavf/rle.gbrapf32le.exr
index 130103e..43db3b6 100644
--- a/tests/ref/lavf/rle.gbrapf32le.exr
+++ b/tests/ref/lavf/rle.gbrapf32le.exr
@@ -1,3 +1 @@
-94398a5ce98bb7b1b78b2b807306f6d7 *tests/data/images/rle.gbrapf32le.exr/02.rle.gbrapf32le.exr
-1436849 tests/data/images/rle.gbrapf32le.exr/02.rle.gbrapf32le.exr
 tests/data/images/rle.gbrapf32le.exr/%02d.rle.gbrapf32le.exr CRC=0x068aca4e
diff --git a/tests/ref/lavf/rle.gbrpf32le.exr b/tests/ref/lavf/rle.gbrpf32le.exr
index f16d710..a10051d 100644
--- a/tests/ref/lavf/rle.gbrpf32le.exr
+++ b/tests/ref/lavf/rle.gbrpf32le.exr
@@ -1,3 +1 @@
-fed878e1f4391314c37088085942e572 *tests/data/images/rle.gbrpf32le.exr/02.rle.gbrpf32le.exr
-1208298 tests/data/images/rle.gbrpf32le.exr/02.rle.gbrpf32le.exr
 tests/data/images/rle.gbrpf32le.exr/%02d.rle.gbrpf32le.exr CRC=0x95e1053f
diff --git a/tests/ref/lavf/rle.grayf32le.exr b/tests/ref/lavf/rle.grayf32le.exr
index 1ac6dfa..5f44e01 100644
--- a/tests/ref/lavf/rle.grayf32le.exr
+++ b/tests/ref/lavf/rle.grayf32le.exr
@@ -1,3 +1 @@
-2841f839cb5aa98bb7aded800dda7cc7 *tests/data/images/rle.grayf32le.exr/02.rle.grayf32le.exr
-410437 tests/data/images/rle.grayf32le.exr/02.rle.grayf32le.exr
 tests/data/images/rle.grayf32le.exr/%02d.rle.grayf32le.exr CRC=0x5443a443
diff --git a/tests/ref/lavf/zip1.gbrapf32le.exr b/tests/ref/lavf/zip1.gbrapf32le.exr
index 27d6b7b..2f17f01 100644
--- a/tests/ref/lavf/zip1.gbrapf32le.exr
+++ b/tests/ref/lavf/zip1.gbrapf32le.exr
@@ -1,3 +1 @@
-9fca73aac1a2e38969bed55929da48b4 *tests/data/images/zip1.gbrapf32le.exr/02.zip1.gbrapf32le.exr
-902782 tests/data/images/zip1.gbrapf32le.exr/02.zip1.gbrapf32le.exr
 tests/data/images/zip1.gbrapf32le.exr/%02d.zip1.gbrapf32le.exr CRC=0x068aca4e
diff --git a/tests/ref/lavf/zip1.gbrpf32le.exr b/tests/ref/lavf/zip1.gbrpf32le.exr
index a85c786..5512aae 100644
--- a/tests/ref/lavf/zip1.gbrpf32le.exr
+++ b/tests/ref/lavf/zip1.gbrpf32le.exr
@@ -1,3 +1 @@
-a405ee4328719c53f97d351b5910e82b *tests/data/images/zip1.gbrpf32le.exr/02.zip1.gbrpf32le.exr
-897319 tests/data/images/zip1.gbrpf32le.exr/02.zip1.gbrpf32le.exr
 tests/data/images/zip1.gbrpf32le.exr/%02d.zip1.gbrpf32le.exr CRC=0x95e1053f
diff --git a/tests/ref/lavf/zip1.grayf32le.exr b/tests/ref/lavf/zip1.grayf32le.exr
index 536deb2..c74f2ae 100644
--- a/tests/ref/lavf/zip1.grayf32le.exr
+++ b/tests/ref/lavf/zip1.grayf32le.exr
@@ -1,3 +1 @@
-76fe15b4e0b735a6318ca273ff52fb58 *tests/data/images/zip1.grayf32le.exr/02.zip1.grayf32le.exr
-91044 tests/data/images/zip1.grayf32le.exr/02.zip1.grayf32le.exr
 tests/data/images/zip1.grayf32le.exr/%02d.zip1.grayf32le.exr CRC=0x5443a443
diff --git a/tests/ref/lavf/zip16.gbrapf32le.exr b/tests/ref/lavf/zip16.gbrapf32le.exr
index 4dd9814..6bfe158 100644
--- a/tests/ref/lavf/zip16.gbrapf32le.exr
+++ b/tests/ref/lavf/zip16.gbrapf32le.exr
@@ -1,3 +1 @@
-8beb8db200e658e74fdd4ed4c4ca9214 *tests/data/images/zip16.gbrapf32le.exr/02.zip16.gbrapf32le.exr
-807513 tests/data/images/zip16.gbrapf32le.exr/02.zip16.gbrapf32le.exr
 tests/data/images/zip16.gbrapf32le.exr/%02d.zip16.gbrapf32le.exr CRC=0x068aca4e
diff --git a/tests/ref/lavf/zip16.gbrpf32le.exr b/tests/ref/lavf/zip16.gbrpf32le.exr
index 7e6f144..7eff2c7 100644
--- a/tests/ref/lavf/zip16.gbrpf32le.exr
+++ b/tests/ref/lavf/zip16.gbrpf32le.exr
@@ -1,3 +1 @@
-4bc7e2640ea086b7fc31cbcd4fff19f5 *tests/data/images/zip16.gbrpf32le.exr/02.zip16.gbrpf32le.exr
-796693 tests/data/images/zip16.gbrpf32le.exr/02.zip16.gbrpf32le.exr
 tests/data/images/zip16.gbrpf32le.exr/%02d.zip16.gbrpf32le.exr CRC=0x95e1053f
diff --git a/tests/ref/lavf/zip16.grayf32le.exr b/tests/ref/lavf/zip16.grayf32le.exr
index da74286..25cb361 100644
--- a/tests/ref/lavf/zip16.grayf32le.exr
+++ b/tests/ref/lavf/zip16.grayf32le.exr
@@ -1,3 +1 @@
-5d5def6a4f16cef1122280d80c7c527d *tests/data/images/zip16.grayf32le.exr/02.zip16.grayf32le.exr
-63997 tests/data/images/zip16.grayf32le.exr/02.zip16.grayf32le.exr
 tests/data/images/zip16.grayf32le.exr/%02d.zip16.grayf32le.exr CRC=0x5443a443
